### PR TITLE
Complete one approval test

### PR DESCRIPTION
**Pull request by hoyjdev**
I'm following this templating tutorial to learn how to write HTML with Go: https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/html-templates#introducing-approval-tests

### CODE REVIEW

**Review by badsketch — APPROVED** [review]

File: web/index_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package web_test
```
Comment (badsketch) [inline web/index_test.go:1]: Is it Go convention to put tests in `<pk being tested>_test`?

File: web/index.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	"io"
+)
+
+type Post struct {
+	Title, Description, Body string
+	Tags                     []string
+}
+
+var (
+	//go:embed "templates/*"
```
Comment (badsketch) [inline web/index.go:15]: Wat this do?

File: web/index.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+func Render(w io.Writer, p Post) error {
+	templ, err := template.ParseFS(postTemplates, "templates/*.gohtml")
+	if err != nil {
+		return err
+	}
+
+	if err := templ.Execute(w, p); err != nil {
+		return err
+	}
```
Comment (badsketch) [inline web/index.go:27]: Interesting, seems like a more "succinct" way to write
```
err := foo()
if err != nil {
  return err
}
```

**Review by badsketch — APPROVED** [review]

File: web/index.go (hunk 2 above)
Comment (hoyjdev) [reply web/index.go:15]: I honestly could not tell you

File: web/index_test.go (hunk 1 above)
Comment (hoyjdev) [reply web/index_test.go:1]: Seems that way

**Pull request merged**